[2024-03-01 09:12:29,310] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: branchstests.gera_numero_aleatorio_task 2024-03-01T09:12:28.113598+00:00 [queued]>
[2024-03-01 09:12:29,334] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: branchstests.gera_numero_aleatorio_task 2024-03-01T09:12:28.113598+00:00 [queued]>
[2024-03-01 09:12:29,334] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-03-01 09:12:29,334] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2024-03-01 09:12:29,334] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-03-01 09:12:29,348] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): gera_numero_aleatorio_task> on 2024-03-01T09:12:28.113598+00:00
[2024-03-01 09:12:29,358] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'branchstests', 'gera_numero_aleatorio_task', '2024-03-01T09:12:28.113598+00:00', '--job-id', '262', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/branchs.py', '--cfg-path', '/tmp/tmpbc4g9u59', '--error-file', '/tmp/tmpcivl7nmy']
[2024-03-01 09:12:29,354] {standard_task_runner.py:52} INFO - Started process 119 to run task
[2024-03-01 09:12:29,359] {standard_task_runner.py:77} INFO - Job 262: Subtask gera_numero_aleatorio_task
[2024-03-01 09:12:29,415] {logging_mixin.py:109} INFO - Running <TaskInstance: branchstests.gera_numero_aleatorio_task 2024-03-01T09:12:28.113598+00:00 [running]> on host b6da4fb5c714
[2024-03-01 09:12:29,496] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=branchstests
AIRFLOW_CTX_TASK_ID=gera_numero_aleatorio_task
AIRFLOW_CTX_EXECUTION_DATE=2024-03-01T09:12:28.113598+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-01T09:12:28.113598+00:00
[2024-03-01 09:12:29,497] {python.py:151} INFO - Done. Returned value was: 74
[2024-03-01 09:12:29,532] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=branchstests, task_id=gera_numero_aleatorio_task, execution_date=20240301T091228, start_date=20240301T091229, end_date=20240301T091229
[2024-03-01 09:12:29,573] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-03-01 09:12:29,613] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
